debug-tests

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+gdb --args build/debug/test/tester ion-tests
```

Comment:
My point is there should be some kind of *TODO*, issue, or event a mention at a minimum that this is specific to a particular platform and we'll support other platforms in another way.

We don't need to support all the platforms, but we should be thinking about what we need to do for the ones we need to support.